conf/modules.config

```diff
@@ -56,30 +56,23 @@ process {
         ]
     }
 
-    withName: '.*FASTQTOBAM' {
+    withName: 'FASTQTOBAM' {
```

Comment:
I believe the `.*FASTQTOBAM` is so this can still be valid if fastquorum is used as a subworkflow/module in other contexts. We should leave the globs as is for now.

Reply:
> I believe the `.*FASTQTOBAM` is so this can still be valid if fastquorum is used as a subworkflow/module in other contexts. We should leave the globs as is for now.

I spent several hours a few months ago trying to debug why my config with 'ALIGN_RAW_BAM' wasn't overriding the current modules.config. It'll always match the name FASTQTOBAM, whether in a subworkflow or not, but this will also match any process including the string "FASTQTOBAM".
You should only use the `.*` if you want to match multiple processes.